<launch>

    <arg name="visual_odom_topic" default="odom" />

    <node name="pengo_stalker_node" pkg="pengo_stalker" type="pengo_stalker_node">
        <remap from="visual_odom" to="$(arg visual_odom_topic)" />
        
        <param name="patrol_points" value="5" />
        <param name="patrol_radius" value="0.5" />
    </node>

</launch>